src/main/scala/org/ergoplatform/explorer/services/AddressesService.scala

```diff
@@ -27,7 +27,13 @@ class AddressesServiceIOImpl[F[_]](xa: Transactor[F], ec: ExecutionContext)
 
   override def getAddressInfo(addressId: String): F[AddressInfo] = for {
```

Comment:
it looks hard to read this `for` fragment. I'm not sure, could it be simplified?